packages/next/next-server/lib/utils.ts

```diff
@@ -207,6 +207,7 @@ export type NextApiResponse<T = any> = ServerResponse & {
    */
   json: Send<T>
   status: (statusCode: number) => NextApiResponse<T>
+  redirect: (url: string, statusCode?: number) => NextApiResponse<T>
```

Comment:
This signature looks wrong, the express documentation clearly shows:

```
res.redirect(status, url)
```

With the option for `status` to also be the url and then the default is the `302` status code